book/website/reproducible-research/testing/testing-guidance.md

```diff
@@ -63,7 +63,6 @@ Find one you like, learn about the features it offers, and make use of them. Com
   - Boost::Test
   - google-test
 - C
-  - all C++ frameworks
```

Comment:
Instead of removing this sentence, we could add a [note](https://the-turing-way.netlify.app/community-handbook/style/style-more-styling.html):

```{note}
While modern C++ and C are still mostly compatible, they're not completely and using test framework interchangeably may not always work.
```